mix.exs

```diff
@@ -28,7 +28,7 @@ defmodule Slax.Mixfile do
   def application do
     [
       mod: {Slax, []},
-      extra_applications: [:logger]
+      extra_applications: [:logger, :quantum]
```

Comment:
Shouldn't need to add this here. It should be done automatically. This is only for extra applications that are outside of your deps

Reply:
gotcha. Want me to take it out?